packages/ag-charts-website/src/content/docs/quick-start/index.mdoc

```diff
@@ -219,26 +266,25 @@ root.render(<ChartExample />);
 
 {% if isFramework("angular") %}
 
-## Create an Angular Chart
-
-Replace your `app.component.ts` file (or root component) with the following:
+**2. Import the Angular Chart**
 
 ```js
 import { Component } from '@angular/core';
+
 import { AgCharts } from 'ag-charts-angular';
 import { AgChartOptions } from 'ag-charts-community';
+```
+
+**3. Define Chart Data and Series**
 
+```jsx
 // Angular Chart Component
 @Component({
   selector: 'app-root',
   standalone: true,
   imports: [AgCharts],
   // ag-charts-angular component with chartOptions attribute
-  template:
-  `<ag-charts
-    style="height: 100%"
-    [options]="chartOptions">
-   </ag-charts>`,
+  template:``,
```

Comment:
Nit: Might be better to just drop this from the example, or add some indicator that the example is glossing over this part?

e.g.
```
template: `<!-- ... -->`
```

Reply:
I've removed the comment from above 'ag-charts-angular component with chartOptions attribute' but left the empty template in in line with the AG Grid angular getting started guide: https://www.ag-grid.com/angular-data-grid/getting-started/ 

Let me know if you're happy with this approach.